tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       displayName: 'Get ONNX testdata'
       inputs:
         filename: azcopy
-        arguments: ' /S /Source:https://onnxruntimetestdata.blob.core.windows.net/onnx-model-zoo-20181018 /Dest:$(Build.SourcesDirectory)\build\Windows\Debug\models /SourceKey:%AZURE_BLOB_KEY%'
+        arguments: ' /S /Source:https://onnxruntimetestdata.blob.core.windows.net/onnx-model-zoo-20181206 /Dest:$(Build.SourcesDirectory)\build\Windows\Debug\models /SourceKey:%AZURE_BLOB_KEY%'
```

Comment:
Why did we need a new version of the models?

Reply:
Removed this dependency now. Reading directly from PB files